src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/cas2/AssessmentsController.kt

```diff
@@ -0,0 +1,46 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.controller.cas2
+
+import org.springframework.http.ResponseEntity
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.cas2.AssessmentsCas2Delegate
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.Cas2Assessment
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.UpdateCas2Assessment
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.problem.BadRequestProblem
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.problem.ConflictProblem
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.problem.ForbiddenProblem
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.problem.NotFoundProblem
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.results.AuthorisableActionResult
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.results.ValidatableActionResult
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.service.cas2.AssessmentService
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.transformer.cas2.AssessmentsTransformer
+
+@Service("Cas2AssessmentsController")
+class AssessmentsController(
+  private val assessmentService: AssessmentService,
+  private val assessmentsTransformer: AssessmentsTransformer,
+) : AssessmentsCas2Delegate {
+
+  override fun assessmentsAssessmentIdPut(
+    assessmentId: java.util.UUID,
+    updateCas2Assessment: UpdateCas2Assessment,
```

Comment:
it's not yet been updated when it's passed in here, so I think I'd rather not use the past tense :)

Reply:
Fair enough, `UpdateCas2Assessment` sounded more verb-like, more like a function.  Was thinking something to make it more noun-like to make it more like an object.

Reply:
I get what you mean but I think in this case as it is pointing to an `UpdateCas2Assessment` model, it makes sense that it would just be the camel-case version of it - we're also following the naming conventions in other controllers like the CAS1 Assessment Controller.
I think if there weren't the precedent to follow I would have called these types of models `AssessmentUpdate` or `ApplicationUpdate` but I think it makes more sense to follow the set patterns now.